lib/fastlane/plugin/notarize/actions/notarize_action.rb

```diff
@@ -53,8 +54,14 @@ def self.run(params)
         while notarization_info.empty? || (notarization_info['Status'] == 'in progress')
           if notarization_info.empty?
             UI.message('Waiting to query request status')
-          else
-            UI.message('Request in progress, waiting to query again')
+          elsif try_early_stapling  
+            UI.message('Request in progress, trying early staple')
+            if self.attempt_staple(package_path)
+              UI.message('Stapled early!  No longer waiting for notarization info.')
+              notarization_info["Status"] = "success"
```

Comment:
Doing this would staple the package twice, right? As an alternative, we can just return here but update the message for the line above to something like this (just to mention that the operation was successful):
> Successfully notarized and early stapled package.